### DISCUSSION TITLE
IA COM PROGRAMAÇÃO

### DISCUSSION THREAD

**Discussion by enricomolarii** [community/community#135923] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Gostaria de saber a opnião da nossa grande comunidade. 
Oque vocês acham de usar IA que trabalha com conceitos que muitas vezes não conhecemos e que aplicamos por ver que funciona no codigo? Seria ruim, aplicar no trabalho algo que não estudamos?

**Comment by LiteBrite82** [comment]
Hi @enricomolarii , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
